static/js/src/cve/cve.js

```diff
@@ -227,27 +179,27 @@ function handleClearFilters() {
 }
 handleClearFilters();
 
-// Maintained releases and vulnerable statuses are handled differently
-// because they are both arrays instead of individual values
+// Maintained releases and vulnerable params statuses are handled differently
+// because they are both arrays of params instead of individual values.
+// Existing params are checked before appending to the URLSearchParams object
+// to avoid duplicates
 function addParam(param, value) {
   if (value === "maintained") {
     maintainedReleases.forEach(function (release) {
-      filterParams.push(
-        { param: param, value: release },
-        { param: "status", value: "" }
-      );
+      if (!urlParams.has(param, release)) {
+        urlParams.append(param, release);
+      }
```

Comment:
We can remove this catch if the all checkboxes are already ticked before submit.